lib/usd/ui/layerEditor/saveLayersDialog.cpp

```diff
@@ -632,13 +635,8 @@ QString SaveLayersDialog::buildTooltipForLayer(SdfLayerRefPtr layer)
 
 QWidget* SaveLayersDialog::findEntry(SdfLayerRefPtr key)
 {
-    if (!_anonLayersWidget || !_anonLayersWidget->layout()) {
-        return nullptr;
-    }
-
-    QLayout* anonLayout = _anonLayersWidget->layout();
-    for (int i = 0, count = anonLayout->count(); i < count; ++i) {
-        auto row = dynamic_cast<SaveLayerPathRow*>(anonLayout->itemAt(i)->widget());
+    for (int i = 0, count = _saveLayerPathRows.size(); i < count; ++i) {
```

Comment:
I kept the order of iteration the same as before, Although in this case it shouldn't matter as it is only picking one item.